portal-web/test/functional/com/liferay/portalweb/macros/SetupWizard.macro

```diff
@@ -137,7 +137,7 @@ definition {
 		else {
 			var baseURL = PropsUtil.get("portal.url");
 
-			Open.openNoError(locator1 = "${baseURL}/web/guest/home");
+			Navigator.openSpecificURL(url = "${baseURL}/web/guest/home");
```

Comment:
Hi @catarineaguiar 

Thanks for working on this! For my understanding, this change is fixing below flaky error in Setup Wizard tests, right?
`java.lang.Exception: An error occurred while executing open`

But seems it doesn't work, a test still failed with this change:
https://testray.liferay.com/home/-/testray/case_results/1955951791

I am not completely sure about why the error occurs, it looks a little tricky. I guess maybe when open home page after restarting portal, the portal is not fully startup, so there is a timeout error. And because there is already a window opened before shutdown portal, so the test thinks the page has already been loaded.

I try with other two different ways in https://github.com/zhangyan0701/liferay-portal/pull/396 and https://github.com/zhangyan0701/liferay-portal/pull/398 and run tests. I don't know which one can fix the error, maybe neither. So I think we can keep this PR open and wait for the results. If one of them can work, I will send to Victor together.

Reply:
Hi Summer, thanks for checking. Waiting for these results :)